bookwyrm/settings.py

```diff
@@ -364,3 +364,6 @@
 OTEL_SERVICE_NAME = env("OTEL_SERVICE_NAME", None)
 
 TWO_FACTOR_LOGIN_MAX_SECONDS = 60
+
+if HTTP_X_FORWARDED_PROTO:
```

Comment:
I think you'll need to add a line creating this variable, something like:
`HTTP_X_FORWARDED_PROTO = env("HTTP_X_FORWARDED_PROTO")`

Reply:
Upside. Of course I need. 

Sorry, I am just getting back to coding after _years_ of not coding :/

Reply:
It's no trouble at all, glad to have your contributions :)